lib/digital_public_works_web/templates/project/_projects.html.eex

```diff
@@ -1,3 +1,6 @@
+<% is_admin = (@conn.assigns[:current_user] != nil) and @conn.assigns[:current_user].is_admin %>
```

Comment:
This could be extracted into a global helper module that could be included in all views.